docs/synapse/deploymentguide.rst

```diff
@@ -0,0 +1,186 @@
+.. toctree::
+    :titlesonly:
+
+.. _deploymentguide:
+
+Deployment Guide
+################
+
+Introduction
+============
+
+This step-by-step guide will walk you through a production-ready Synapse deployment. Services will be
+configured register with ``AHA`` for service discovery and to prepare for future devops tasks such
+as promoting a mirror to leader and registering ``Synapse Advanced Power-Ups``.
+
+This guide will also walk you through deploying all Synapse services using TLS to authenticate both
+servers and clients using client-certificates to minimize the need for secrets management by eliminating
+passwords from all deployment configurations.
+
+Deployment FAQ
+--------------
+
+Conventions
+===========
+
+* Docker-Compose
+
+For the purposes of this guide, we will use ``docker-compose`` as a light-weight orchestration mechanism.
+The steps, configurations, and volume mapping guidance given in this guide apply equally to other container
+orchestration mechanisms such as ``Kubernetes`` but for simplicity's sake, this guide will only cover
+``docker-compose`` based deployments.
+
+* Container Users
+
+In order to run the ``Synapse`` service containers as a non-root user, you will need to add a user
+to the host system.  For this guide, we will use the linux user name ``synuser``. This user name can be replaced
+by whatever user name or numeric ID is appropriate for your deployment. We recommend that you do *not* use the
+linux user ``nobody`` for this purpose.
```

Comment:
`nobody` is a optional user that isn't guaranteed to be present in any linux hosts. could probably omit that reference.

Reply:
Added this reference as a result of a specific question by a customer ( and it's a standard user on almost all linux distributions )